tests/runtime/call

```diff
@@ -229,7 +229,7 @@ AFTER ./testprogs/uprobe_loop
 NAME ustack_stack_mode_env_bpftrace
 PROG u:./testprogs/uprobe_loop:uprobeFunction1 { printf("%s\n", ustack(1)); exit(); }
 ENV BPFTRACE_STACK_MODE=bpftrace
-EXPECT uprobeFunction1+0
+EXPECT_REGEX uprobeFunction1\+[0-9]+
```

Comment:
```suggestion
EXPECT_REGEX ^uprobeFunction1\+[0-9]+$
```

Part of the reason `EXPECT_REGEX` is dangerous is that it doesn't enforce a full line match by default. A partial line match here would match other results in addition to what we want, e.g. a stack trace in "perf" format